### DISCUSSION TITLE
Codespaces v2 connection results in network error

### DISCUSSION THREAD

**Discussion by kbdharun** [community/community#109419] [Codespaces] [answered]
### Select Topic Area

Product Feedback

### Body

Since the announcement at [GitHub blog](https://github.blog/changelog/2024-02-12-enhanced-codespaces-connection/), I have been trying out the new Codespaces connection but every time it results in a network error when I use it in my university network (it works fine at home). I can access `visualstudio.com` just fine so not sure why newer connections fail. If I disable the feature preview and use the old codespace connection, it works fine.

![image](https://github.com/community/community/assets/26346867/affb185f-9420-46ca-b45f-8d322e50d1f7)

I don't think it is due to any firewall blocks (for context my university uses **Fortinet**). Is there any way to fix this issue before the migration?

**Edit.** Additional details about the infrastructure, can be found [here](https://soc.sastra.edu/index.php/infrastructure.html). I will attach the network specs below.

![image](https://github.com/community/community/assets/26346867/b860995e-7abb-4ed2-93a3-61ce88f63263)

**Comment by onePlaceholder** [comment]
I have same issue... in my highschool

**Reply by onePlaceholder** [reply]
they also use **Fortinet**

**Reply by kbdharun** [reply]
@someElseIsHere As a temporary workaround, currently you can disable it from **Feature Preview** panel and it will work. Let's hope this issue get's addressed before the wider migration.

![image](https://github.com/community/community/assets/26346867/89c20872-d868-41dd-a3ec-835cf7b244ac)

**Reply by onePlaceholder** [reply]
thank you so much I've been looking for this :)

**Comment by osortega** [comment, marked as answer]
Hello @someElseIsHere @kbdharun !

Sorry for the friction this is causing you.
The description here seems to indicate that the request is being blocked by the school's firewall/network.

Could you please work with them to make sure request to `*.visualstudio.com` are allowed-listed?

**Reply by kbdharun** [reply]
Thanks for the reply, I will try to contact them to see if it is allow listed. If I am not able to get this resolved IG I will use Codespaces via Cellular or use Gitpod instead (under the university's network).

**Comment by AiZeroToOne** [comment]
me to
